tests/h/services/group_test.py

```diff
@@ -175,6 +175,17 @@ def test_annotation_count_returns_count_of_shared_annotations_for_group(self, db
 
         assert svc.annotation_count(group.pubid) == 3
 
+    def test_world_readable_pubids(self, db_session, factories, users):
+        # Groups that are members readable - should not be returned
+        [factories.Group(readable_by=ReadableBy.members) for _ in xrange(2)]
+        # Groups that are world readable - should be returned
+        world_readable = [factories.Group(readable_by=ReadableBy.world) for _ in xrange(3)]
```

Comment:
I guess I can import `h._compat.xrange`, but doesn't this imply that Hound runs flake8 with Python 3?